docs/api/security/crypto.md

```diff
@@ -0,0 +1,49 @@
+## Crypto
+
+Arm Mbed Crypto is the reference implementation of the cryptography interface
+of the Arm Platform Security Architecture (PSA). This is a preview release of
```

Comment:
What is a preview release of Mbed Crypto? 5.12? This document? Please clarify.

I ask because I want this document to be maintainable. If this statement will change by the next feature release, we should remove it.

Reply:
There is no stable PSA Crypto API yet, so as the PSA Crypto API changes (somewhat out of our control), we need to adapt and follow suit in order to pass compliance tests. This may mean API breaks for our users, hence the warning. I'm open to suggestions as to how to word this better.